apps/server/src/modules/learnroom/controller/api-test/rooms-copy-timeout.api.spec.ts

```diff
@@ -22,7 +21,9 @@ Configuration.set('INCOMING_REQUEST_TIMEOUT_COPY_API', 1);
 // eslint-disable-next-line import/first
 import { ServerTestModule } from '@src/modules/server/server.module';
 
-describe('Copy timeout (e2e)', () => {
+// This needs to be in a separate test file because of the above configuration.
+// When we find a way to mock the config, it should be moved alongside the other API tests.
+describe('Rooms copy (API)', () => {
```

Comment:
Irgendwie ist das Naming in diesem Bereich nicht ganz rund oder? learnroom, room**s** und course? Aber das ist wahrscheinlich ein anderes Thema.

Und warum sind die room-Files im plural benannt?

Reply:
Ist mir auch aufgefallen, aber das gerade zu ziehen wär zu viel für dieses Ticket, denke ich.